src/web/handlers.rs

```diff
@@ -193,7 +193,7 @@ fn content_response(content: &cache::TilesContent) -> HttpResponse {
     match content {
         cache::TilesContent::Json(json) => HttpResponse::Ok()
             .content_type("application/json")
-            .body(json),
+            .body(json.to_owned()),
```

Comment:
I'll note that actix-web has always internally made a copy of data like `&str` here for response bodies. actix-web 4's API just exposes that to the user, requiring them to make the copy. This makes sense with responses being used in asynchronously, a lifetime would complicate that.

However a `&'static str` can be easily used asynchronously so it supports that and we utilize it w/ `body(EMPTY_TILES.as_str())` (no copying) above.